bin/_docker.sh

```diff
@@ -69,6 +72,8 @@ docker_build() {
 
     if [ "$DOCKER_BUILDKIT_CACHE" ]; then
       cache_params="--cache-from type=local,src=${DOCKER_BUILDKIT_CACHE} --cache-to type=local,dest=${DOCKER_BUILDKIT_CACHE},mode=max"
+    elif [ "$ACTIONS_CACHE_URL" ]; then
```

Comment:
The condition above (`$DOCKER_BUILDKIT_CACHE`) might be fine to clean? Doing a grep through the repo, I can only find references to it in:
* The build action (which has been cleaned up)
* This script
* The prune script (which we no longer call)

...so it miiiiight be safe to rm?

```
:; rg 'DOCKER_BUILDKIT_CACHE'
bin/docker-cache-prune
8:export DOCKER_BUILDKIT_CACHE=${DOCKER_BUILDKIT_CACHE:-}
10:manifest_sha=$(jq -r .manifests[0].digest < "${DOCKER_BUILDKIT_CACHE}/index.json")
13:while IFS= read -r line; do files+=("$line"); done <<< "$(jq -r '.manifests[].digest | sub("^sha256:"; "")' < "${DOCKER_BUILDKIT_CACHE}/blobs/sha256/$manifest")"
14:for file in "${DOCKER_BUILDKIT_CACHE}"/blobs/sha256/*; do

bin/_docker.sh
19:export DOCKER_BUILDKIT_CACHE=${DOCKER_BUILDKIT_CACHE:-}
70:    if [ "$DOCKER_BUILDKIT_CACHE" ]; then
71:      cache_params="--cache-from type=local,src=${DOCKER_BUILDKIT_CACHE} --cache-to type=local,dest=${DOCKER_BUILDKIT_CACHE},mode=max"
```

Reply:
yup happy to remove, relates to my other comment.